examples/openai/token-metadata.php

```diff
@@ -23,18 +24,26 @@
     'temperature' => 0.5, // default options for the model
 ]);
 
-$agent = new Agent($platform, $model, outputProcessors: [new TokenOutputProcessor()], logger: logger());
+$agent = new Agent(
+    $platform,
+    $model,
+    outputProcessors: [new TokenUsageOutputProcessor(new TokenUsageExtractor())],
+    logger: logger()
+);
 $messages = new MessageBag(
     Message::forSystem('You are a pirate and you write funny.'),
     Message::ofUser('What is the Symfony framework?'),
 );
+
 $result = $agent->call($messages, [
     'max_tokens' => 500, // specific options just for this call
 ]);
 
-$metadata = $result->getMetadata();
+if (null === $tokenUsage = $result->getTokenUsage()) {
```

Comment:
Can this happen?

Reply:
Yes, if you mean `null === $tokenUsage`, when _token_usage_ is marked false in the config, as far as I can see.